configs/example/hmc_hello.py

```diff
@@ -68,10 +67,13 @@
 # connect special port in the system to the membus. This port is a
 # functional-only port to allow the system to read and write memory.
 system.system_port = system.membus.cpu_side_ports
-# get ISA for the binary to run.
-isa = get_runtime_isa()
 # run 'hello' and use the compiled ISA to find the binary
-binary = "tests/test-progs/hello/bin/" + isa.name.lower() + "/linux/hello"
+
+
+# binary = (
+#    "tests/test-progs/hello/bin/" + options.isa.name.lower() + "/linux/hello"
+# )
```

Comment:
Here is a problem if we remove the target ISA.

We could change this so we pass the binary as a parameter in this case. Is that ok?